src/components/safe-apps/SafeAppList.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import Grid from '@mui/material/Grid'
+import Typography from '@mui/material/Typography'
+import { SafeAppsSection } from './SafeAppsSection'
+import { useAppsSearch } from '@/hooks/safe-apps/useAppsSearch'
+import { useSafeApps } from '@/hooks/safe-apps/useSafeApps'
+import { SafeAppsHeader } from './SafeAppsHeader'
+
+const SafeAppList = () => {
+  const [searchQuery, setSearchQuery] = React.useState('')
+  const {
+    allSafeApps,
+    pinnedSafeApps,
+    pinnedSafeAppIds,
+    remoteSafeAppsLoading,
+    customSafeAppsLoading,
+    addCustomApp,
+    customSafeApps,
+    togglePin,
+  } = useSafeApps()
+  const filteredApps = useAppsSearch(allSafeApps, searchQuery)
+
+  let pageBody = (
+    <>
+      <SafeAppsSection
+        collapsible
+        title={`PINNED APPS (${pinnedSafeApps.length})`}
```

Comment:
Why is it caps lock? Please use `text-transform: uppercase` if it's for design reasons.